### PR TITLE
[upgrade-automation]: Test suite for upgrading from 5.3 to 6.0 latest

### PR DESCRIPTION
**Pull request by anrao19**
Signed-off-by: Anuchaithra Rao <anrao@redhat.com>

# Description
PR includes 
1. rename of multisite suite which perform upgrade from 5.2GA to 6.0
2. upgrade from 5.3 to 6.0  Single site suite

LOG: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/cephci-run-LB2P8B/
issue seen after upgrade is due to bucket creation failure 
we have bz for that : https://bugzilla.redhat.com/show_bug.cgi?id=2156482

Latest log: http://magna002.ceph.redhat.com/ceph-qe-logs/Anuchaithra/cephci-run-3B96H8/

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                custom_image: "registry-proxy.engineering.redhat.com/rh-osbs/rhceph:ceph-5.3-rhel-8-containers-candidate-96954-20221222232354"
+                custom_repo: "http://download.eng.bos.redhat.com/rhel-8/composes/auto/ceph-5.3-rhel-8/RHCEPH-5.3-RHEL-8-20221222.ci.2/compose/Tools/x86_64/os/"
```
Comment (udaysk23) [inline suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml:23]: Repos may become inaccessible after certail days. Wont it fail in that case?

File: suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+      config:
+        script-name: test_bucket_listing.py
+        config-file-name: test_bucket_radoslist.yaml
+        timeout: 300
+      desc: Test bucket stats and radoslist on all buckets
+      module: sanity_rgw.py
+      name: test bucket stats and radoslist
+      polarion-id: CEPH-83574480
+
+# Object download test
```
Comment (udaysk23) [inline suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml:170]: Please remove or correct this line

Comment (Ameenasuhani) [inline suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml:170]: This is a upgrade test right can we add a appropriate comment?

File: suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+  - test:
+      name: Upgrade ceph
+      desc: Upgrade cluster to latest version
+      module: cephadm.test_cephadm_upgrade.py
```
Comment (Ameenasuhani) [inline suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml:174]: Just curious to know, conventionally we use **module: test_cephadm_upgrade.py** any reason why we are using **module: cephadm.test_cephadm_upgrade.py**

File: suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml [hunk 4]
```diff
@@ -0,0 +1,284 @@
+#Polarian ID -CEPH-83574647
+#Objective: Testing single site upgrade from RHCS 5.3 latest to 6.x latest build
+#platform : RHEL-9
+#conf: tier-0_rgw.yaml
```
Comment (Ameenasuhani) [inline suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml:4]: Can we have full conf path mentioned here?

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**